sdk/python/resources/workspace/workspace-managed-network.ipynb

```diff
@@ -0,0 +1,366 @@
+   "source": [
+    "ws_mvnet = ml_client.workspaces.get(mvnet_workspace_name)\n",
+    "\n",
+    "ws_mvnet.managed_network.outbound_rules.append(\n",
+    "    ServiceTagDestination(\n",
+    "        name=\"servicetagrule\",\n",
+    "        service_tag=\"AzureCloud\",\n",
+    "        protocol=\"TCP\",\n",
+    "        port_ranges=\"80, 8080\",\n",
+    "    )\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3.2 Add a FQDN outbound rule\n",
+    "Note: FQDN outbound rules are implemented using Azure Firewall. If you use outbound FQDN rules, charges for Azure Firewall are included in your billing.\n",
+    "\n",
+    "To create a FQDN outbound rule, we will define the `FqdnDestination` Python object with the following attributes:\n",
+    "- `name` - Name of the FQDN outbound rule.\n",
+    "- `destination` - Fully qualified domain name to which outbound connections are allowed, for example: “*.contoso.com”."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ws_mvnet.managed_network.outbound_rules.append(\n",
+    "    FqdnDestination(name=\"fqdnrule\", destination=\"pypi.org\")\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3.3 Add a PrivateEndpoint outbound rule\n",
+    "To create a PrivateEndpoint outbound rule, we will define the `PrivateEndpointDestination` Python object with the following attributes:\n",
+    "- `name` - Name of the ServiceTag outbound rule.\n",
+    "- `service_resource_id` - The resource URI (ARM ID) of the root service that supports creation of the private link.\n",
+    "- `subresource_target` - The subresource type for the service for the private link resource.\n",
+    "- `spark_enabled` - (Optional) Indicates if the private endpoint can be used for Spark jobs, default is “false”.\n",
+    "\n",
+    "Refer to [AzureML Managed Network documentation](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-managed-network#private-endpoints) for the currently supported Private Enpoint destinations. \n",
+    "\n",
+    "Refer to [Private Link documentation](https://learn.microsoft.com/en-us/azure/private-link/private-endpoint-overview#private-link-resource) for the possible sub resource targets for each resource provider.\n",
+    "\n",
+    "Note: For simplicty of this demonstration, we will add a `table` Private Endpoint outbound rule to the workspace-associated storage account. (also note that file and blob are already default required rules on that storage account)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ws_mvnet.managed_network.outbound_rules.append(\n",
+    "    PrivateEndpointDestination(\n",
+    "        name=\"privateendpointrule\",\n",
+    "        service_resource_id=ws_mvnet.storage_account,\n",
```

Comment:
If the customer run this example, it will get an error since the rule to this destination already exists since they switched to AOAO. maybe just use a fake ID

Reply:
it was leftover from my testing. it should be 

subscription_id = "<SUBSCRIPTION_ID>"
resource_group = "<RESOURCE_GROUP>"

since its run against CI